client/src/pages/Profile.js

```diff
@@ -105,12 +119,13 @@ export default function Profile({ authProps, setAuthProps }) {
         toast.error(err.message);
       } else {
         await axios
-          .put(`/api/user/put?user_email=${authProps.user.user_email}`, {
+          .put(`http://localhost:8000/api/user/put?user_email=${authProps.user.user_email}`, {
```

Comment:
this api link will have to be adjusted for the deployed heroku server api

Reply:
the adjusted api link for the deployed heroku link: https://nutrimental-server.herokuapp.com/api....